metro-ai-suite/metro-vision-ai-app-recipe/smart-intersection/src/dlstreamer-pipeline-server/config.json

```diff
@@ -280,6 +280,286 @@
                         }
                     }
                 }
+            },
+            {
+                "name": "intersection-cam1_gpu",
```

Comment:
Ideally, let's rename the first four pipelines to have the _cpu suffix to be consistent.